src/pages/Add-items/Add-items.js

```diff
@@ -127,6 +133,7 @@ export const AddItems = () => {
                 name="howSoon"
                 required
                 value={30}
+                checked={product.howSoon === '30'}
                 onChange={handleChangeProduct}
               />
               Not soon
```

Comment:
Tengo una duda, cuando agregamos un nuevo ítem,  en el firebase, en el elemento howSoon me sale un numero de días dependiendo de lo que elegimos (Soon, Kind of soon, Not soon), es cuando lo checkeamos por primera vez que este cambia a 0. Es necesario mostrar este numero apenas se agregue el ítem ?

Reply:
¡Excelente pregunta Dara! 👏🏻 

La respuesta es: de acuerdo a lo que pedía el Issue #4 sí es necesario PERO según lo que hace la función `calculateEstimate` no tiene ningún efecto: sin importar cuál sea el valor de `howSoon`, ésta va a devolver 0 la primera vez que hay una compra entonces el valor inicial de `howSoon` no afecta nada.

Esto podría significar o que entendimos mal algo de cuáles son los valores que se le deben pasar a la función o que hay algo que ajustarle.


Esta es la fn en detalle para que nos ayuden a evaluarlo:

```javascript
const calculateEstimate = (previousEstimate = 14, // The last estimated purchase interval
daysSinceLastTransaction, // The number of days since the item was added to the list or last purchased
totalPurchases // Total number of purchases for the item
) => {
    // Not enough data if an item has been purchased 1 time,
    // just set the estimate based on when it was added to the list
    if (totalPurchases < 2)
        return daysSinceLastTransaction;
    // This calculates how many days should have passed based on
    // the previous estimate between purchases and the total number of purchased
    const previousFactor = previousEstimate * totalPurchases;
    // This calculates how many days should have passed based on
    // the interval between the most recent transactions
    // Subtract 1 here to exclude the current purchase in this factor
    const latestFactor = daysSinceLastTransaction * (totalPurchases - 1);
    // Divisor is used to find the average between the two factors
    // Multiplied by 2 between we will add 2 factors together
    // Subtract 1 here to lower weight of the current purchase in this factor
    const totalDivisor = totalPurchases * 2 - 1;
    //Calculate the average interval between the previous factor and the latest factor
    return Math.round((previousFactor + latestFactor) / totalDivisor);
};
```